clusterloader2/testing/density/100_nodes/constraints.yaml

```diff
@@ -18,7 +18,7 @@ kube-proxy:
   memoryConstraint: 31457280 #30 * (1024 * 1024)
 kube-scheduler:
   cpuConstraint: 0.35
-  memoryConstraint: 115343360 #110 * (1024 * 1024)
+  memoryConstraint: 120586240 #115 * (1024 * 1024)
```

Comment:
Can we bup it a bit more aggresively - bumping by 5MB doesn't make much sense.
I told that couple weeks ago already and i tseems we're still suffering from that.

Reply:
Ok. I changed it to 120MB. It is a ~10% change.

> I told that couple weeks ago already and i tseems we're still suffering from that.

I will not agree with word "suffering". The average usage of memory increased, the test responded with flakes. It drew our attention, so we are aware of the changes made to the kubernetes. From my point of view, it's a good thing.

Reply:
Did it result in any action item? If not, then it's "suffering".